### PR TITLE
remove vendored service module and added features

### PR DESCRIPTION
**Pull request by angstwad**
This PR removes `must_exist` usage to comply with upstream module and allows removal of vendored service module, which is _also_ removed.

### CODE REVIEW

**Comment by scoop206** [comment]
is must_exist a part of this too ?

**Comment by scoop206** [comment]
:+1: but I'm thinking we should hold off on merging this until we get centos in.
centos removes the log stash forwarder usage to it will be easier to rebase this guy against master at that point.

**Comment by angstwad** [comment]
In our pinned version of the service module:

Includes restarted_if_running logic
Includes must_exist
Includes dumping of additional info to syslog via syslogging argument
we're missing out on dramatic re-writes of LinuxService class

## restarted_if_running
appears to just be an extra option whose only purpose is to provide an output state of restarted

File: playbooks/adjust-cinder-backend.yml [hunk 1]
```diff
@@ -223,7 +224,8 @@
     service:
       name: "{{ item[1] }}"
       state: restarted
-      must_exist: false
+    register: result
```
Comment (chengtcli) [inline playbooks/adjust-cinder-backend.yml]: Do we need this register? Seems we don't have a reference to this.

**Comment by nirajdp76** [comment]
retest

**Pull request closed without merging**